pom.xml

```diff
@@ -169,7 +169,7 @@
 		<repository>
 			<id>fuse-early-access</id>
 			<name>Fuse Early Access - staging</name>
-			<url>https://origin-repository.jboss.org/nexus/content/</url>
+			<url>https://repository.jboss.org/nexus/content/</url>
```

Comment:
not related to this PR but the name is not corresponding, this is not the early access repo

Reply:
you are right

Reply:
then you should fix that and resubmit the PR! :D